tests/e2e/tests/critical-path/browser/database-overview.e2e.ts

```diff
@@ -93,59 +97,86 @@ test('Verify that when user adds or deletes a new key, info in DB header is upda
-});
-test('Verify that user can see total memory rounded in format B, KB, MB, GB, TB in DB header in Browser page', async t => {
-    //Connect to DB
-    await myRedisDatabasePage.clickOnDBByName(ossStandaloneConfig.databaseName);
-    //Add new keys
-    await cliPage.addKeysFromCli('MSET', 100);
-    //Verify total memory
-    await t.wait(fiveSecondsTimeout);
-    await t.expect(browserPage.overviewTotalMemory.textContent).contains('MB', 'Total memory value is MB');
-});
-test('Verify that user can see additional information in Overview: Connected Clients, Commands/Sec, CPU (%) using Standalone DB connection type', async t => {
-    //Connect to DB
-    await myRedisDatabasePage.clickOnDBByName(ossStandaloneConfig.databaseName);
-    const cpuBeforeEdit = await browserPage.overviewCpu.textContent;
-    const commandsSecBeforeEdit = await browserPage.overviewCommandsSec.textContent;
-    //Verify that additional information in Overview: Connected Clients, Commands/Sec, CPU (%) is displayed
-    await t.expect(browserPage.overviewConnectedClients.visible).ok('Connected Clients is dispalyed in the Overview');
-    await t.expect(browserPage.overviewCommandsSec.visible).ok('Commands/Sec is dispalyed in the Overview');
-    await t.expect(browserPage.overviewCpu.visible).ok('CPU (%) is dispalyed in the Overview');
-    //Add 1M keys
-    for(let i = 0; i < 10; i++) {
-        await cliPage.addKeysFromCli('MSET', 100000, `keyName${i}`);
-    }
-    //Verify that CPU and commands per second parameters are changed
-    const cpuAfterEdit = await browserPage.overviewCpu.textContent;
-    const commandsSecAfterEdit = await browserPage.overviewCommandsSec.textContent;
-
-    await t.expect(cpuAfterEdit).notEql(cpuBeforeEdit, 'CPU parameter is changed');
-    await t.expect(commandsSecAfterEdit).notEql(commandsSecBeforeEdit, 'Commands per second parameter is changed');
-});
+test
+    .after(async t => {
+        //Clear and delete database
+        await t.click(myRedisDatabasePage.myRedisDBButton);
+        await myRedisDatabasePage.clickOnDBByName(ossStandaloneConfig.databaseName);
+        await cliPage.sendCommandInCli(`DEL ${keys1.join(' ')}`);
+        await cliPage.sendCommandInCli(`DEL ${keys2.join(' ')}`);
+        await deleteDatabase(ossStandaloneConfig.databaseName);
+        await deleteDatabase(ossStandaloneBigConfig.databaseName);
+    })
+    ('Verify that user can see total number of keys rounded in format 100, 1K, 1M, 1B in DB header in Browser page', async t => {
+        //Add 100 keys
+        keys1 = await common.createArrayWithKeyValue(100);
+        await cliPage.sendCommandInCli(`MSET ${keys1.join(' ')}`);
+        //Wait 5 seconds
+        await t.wait(fiveSecondsTimeout);
+        //Verify that the info on DB header is updated after adds
+        let totalKeys = await browserPage.overviewTotalKeys.innerText;
+        await t.expect(totalKeys).eql('100', 'Info in DB header after ADD 100 keys');
```

Comment:
Seems you repeat this action in the test. Please create a function

Reply:
done